examples/basic-https-listener-client/basic_https_listener_client.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on how the Ballerina https client can be configured to connect to an https listener through 1-way SSL connection(i.e.,the server is verified by the client).
```

Comment:
Edit: BBE on how the Ballerina HTTPS client can be configured to connect to an HTTPS listener through 1-way SSL connection (i.e. the server is verified by the client)

(Added correct spacing, removed some punctuation to keep within the character limit)